Dockerfile-acceptance

```diff
@@ -7,6 +7,6 @@ COPY codecept.conf.js codecept.conf.js
 COPY apps apps/
 COPY test.sh test.sh
 
-RUN npm --loglevel warn install --ignore-scripts
+RUN npm --loglevel warn install
 
```

Comment:
My bad i just noticed this is on the acceptance. On the main docker file we do this to speed up docker builds (caching) on acceptance I give 0 fs